components/ListView/ComponentInputs.js

```diff
@@ -1,16 +1,16 @@
 /* global $ */
 
 import React from 'react';
-import {defineMessages, injectIntl, intlShape} from 'react-intl';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
 import PropTypes from 'prop-types';
 import ComponentTypeIcons from '../../components/ListView/ComponentTypeIcons';
 
 const messages = defineMessages({
   hideDetails: {
-    defaultMessage: "Hide Details"
+    defaultMessage: 'Hide Details'
```

Comment:
Note that translatable strings must be marked with `_("...")`, and for xgettext it *has* to be double quotes. This PR doesn't make things any worse, but this goes into the wrong direction.